.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
  - docker
 
 env:
- - DEPENDENCY_PACKAGES="cmake libaccounts-glib-dev libclutter-gtk-1.0-dev libdbusmenu-glib-dev libdbusmenu-gtk3-dev libdbus-glib-1-dev libindicate-dev libgda-5.0-dev libgee-0.8-dev libglib2.0-dev libgpod-dev libgranite-dev libgsignon-glib-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgtk-3-dev libjson-glib-dev libnotify-dev libpeas-dev libsoup2.4-dev libtagc0-dev libwebkitgtk-dev libxml2-dev libzeitgeist-2.0-dev valac"
+ - DEPENDENCY_PACKAGES="cmake libaccounts-glib-dev libclutter-gtk-1.0-dev libdbusmenu-glib-dev libdbusmenu-gtk3-dev libdbus-glib-1-dev libindicate-dev libgda-5.0-dev libgee-0.8-dev libglib2.0-dev libgranite-dev libgsignon-glib-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgtk-3-dev libjson-glib-dev libnotify-dev libpeas-dev libsoup2.4-dev libtagc0-dev libxml2-dev libzeitgeist-2.0-dev valac"
```

Comment:
I think we don't need json-glib-1.0, libsoup-2.4, libaccounts-glib and libgsignon-glib anymore, they were only used by the Last.fm plugin.

Reply:
And it would be nice to remove these packages from the README too.